README.md

```diff
@@ -51,3 +51,9 @@ Please have a look at the [Contribution Guide][contrib] on how to get started.
 [conscribo]: https://www.conscribo.nl/api/
 [tailwind]: https://tailwindcss.com
 [contrib]: ./CONTRIBUTING.md
+
+
+## testing the gumbot
+Via @BotFather you can create a new telegram bot
```

Comment:
Mentions on GitHub will poke and link a GitHub account, that's probably not intended.

```suggestion
Via [@BotFather](https://t.me/BotFather) you can create a new telegram bot
```